### PR TITLE
Add/helper rest api wp org plugins

### PR DESCRIPTION
**Pull request by tgglv**
PR added the last piece for WP.org plugin support into the existing branch.

### CODE REVIEW

File: includes/admin/helper/class-wc-helper-product-install.php [hunk 1]
```diff
@@ -303,44 +319,104 @@ private static function move_product( $product_id, $upgrader ) {
...
+	}
+
+	/**
+	 * Get plugin's relative path.
+	 *
+	 * @param string $folder Folder of the plugin.
+	 *
+	 * @return bool|string
+	 */
+	private static function get_wporg_plugin_relative_path( $folder ) {
```
Comment (gedex) [inline includes/admin/helper/class-wc-helper-product-install.php]: as the output returns plugin main file, how about naming it `get_wporg_plugin_main_file`?

File: includes/admin/helper/class-wc-helper-product-install.php [hunk 2]
```diff
@@ -303,44 +319,104 @@ private static function move_product( $product_id, $upgrader ) {
...
 	}
+
+	/**
+	 * Get WP.org product filename.
+	 *
+	 * @param int $product_id Product ID.
+	 *
+	 * @return bool|string
+	 */
+	private static function get_wporg_product_folder_name( $product_id ) {
```
Comment (gedex) [inline includes/admin/helper/class-wc-helper-product-install.php]: prolly it's just me being pedantic. I think `dir` is more fit than `folder`. Not a biggie though.

**Review by gedex — APPROVED** [review]

File: includes/admin/helper/class-wc-helper-product-install.php (hunk 1 above)
Comment (tgglv) [reply includes/admin/helper/class-wc-helper-product-install.php]: Method name was changed.

File: includes/admin/helper/class-wc-helper-product-install.php (hunk 2 above)
Comment (tgglv) [reply includes/admin/helper/class-wc-helper-product-install.php]: Method name was changed too. Thanks for your suggestion. The new name really suites better!

**Pull request merged**